### PR TITLE
styled-system: Fix spelling of Vertical

### PR DESCRIPTION
**Pull request by jschuler**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: [styled-system](https://github.com/jxnblk/styled-system/blob/e665fc962e9f3280c0d4f7e7a4ad265139a45d34/src/index.js#L390)
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/styled-system/index.d.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@
 //                 Eloy Durán <https://github.com/alloy>
 //                 Matthieu Vachon <https://github.com/maoueh>
 //                 Adam Lavin <https://github.com/lavoaster>
+//                 Joachim Schuler <https://github.com/jschuler>
```
Comment (alloy) [inline types/styled-system/index.d.ts]: Just want to check if the implication of adding to this header is clear? I.e. you’re signing up as a maintainer if the types and will get pinged for each new issue/PR for these types.

**Review by alloy — APPROVED** [review]
LGTM 👍 Thanks!

File: types/styled-system/index.d.ts (hunk 1 above)
Comment (jschuler) [reply types/styled-system/index.d.ts]: Yes thanks, I'd like to help maintain these types

Comment (alloy) [reply types/styled-system/index.d.ts]: Ace 🙏

**Pull request merged**